README.md

```diff
@@ -161,7 +162,7 @@ We are looking for maintainers to add more parsers and to write query files for
 - [x] [gomod](https://github.com/camdencheek/tree-sitter-go-mod) (maintained by @camdencheek)
 - [x] [graphql](https://github.com/bkegley/tree-sitter-graphql) (maintained by @bkegley)
 - [ ] [haskell](https://github.com/tree-sitter/tree-sitter-haskell)
-- [x] [hcl](https://github.com/MichaHoffmann/tree-sitter-hcl) (maintained by @MichaHoffmann)
+- [ ] [hcl](https://github.com/MichaHoffmann/tree-sitter-hcl)
```

Comment:
@MichaHoffmann you need to set yourself in `parser.lua` as a maintainer or CI will always try to delete you

Reply:
Oh, sorry, i did not know that; should i do it in a separate PR?

Reply:
https://github.com/nvim-treesitter/nvim-treesitter/pull/1476